### ISSUE TITLE
Gradient descent based planning versus CEM based planning

### ISSUE THREAD

**Issue by viv92** [lucas-maes/le-wm#65]
Hi,

Thanks for your great work!

I'm curious about your thoughts on using gradient descent for optimizing action sequences during planning over the Cross Entropy Method discussed in the paper. I'd expect gradient descent based optimization to be more efficient as it directly leverages the gradients from the differentiable world model. Moreover, better temporal straightness of latent paths of Le-WM compared to other baselines such as PLDM and Dino-WM will be leveraged by the gradient descent based optimization.

Let me know if you already have some results on this that can be shared.

Thanks
